module/index.js

```diff
@@ -315,6 +315,7 @@ var ClassicAPI = (function () {
     ClassicAPI.prototype.parseURL = function (unparsedURL) {
         var parsedURL = unparsedURL.replace(/{region}/g, region_e_TO_string(this.region));
         parsedURL = parsedURL.replace(/{endpoint}/g, region_e_TO_endpointString(this.region));
+        console.log(parsedURL);
```

Comment:
Not sure where this `console.log` came from sorry about that. I did build from the `index.ts` file.